### ISSUE TITLE
Ftp: build for Scala 2.13

### ISSUE THREAD

**Issue by 2m** [akka/alpakka#1532]
This is a spinoff ticket from #828 that tracks FTP connector cross-building for Scala 2.13.

Currently it is disabled, because of same compilation errors as migrating from 2.12.7 to 2.12.8.

See https://github.com/scala/bug/issues/11305

**Comment by ennru** [comment]
This should be solved in Scala 2.13.0-RC2. https://github.com/akka/alpakka/issues/1719

**Comment by ennru** [comment]
With 2.13.0 compilation fails like this 
```
[error] /home/travis/build/akka/alpakka/ftp/src/test/java/akka/stream/alpakka/ftp/KeyFileSftpSourceTest.java:48:1: incompatible types: java.lang.Object cannot be converted to akka.stream.alpakka.ftp.FtpFile
[error]     return Sftp.move(f -> ROOT_PATH + destinationPath.apply(f), settings());
```

https://travis-ci.com/akka/alpakka/jobs/209169430#L2002

**Comment by ennru** [comment]
Alpakka FTP is available for Scala 2.13 since #1779.
